libs/datasets/timeseries.py

```diff
@@ -126,13 +136,28 @@ class OneRegionTimeseriesDataset:
 
     latest: Dict[str, Any]
 
-    # A default exists for convenience in tests. Non-test is expected to explicitly set provenance.
-    # Because these objects are frozen it /might/ be safe to use default={} but using a factory to
-    # make a new instance of the mutable {} is safer.
-    provenance: Dict[str, str] = dataclasses.field(default_factory=dict)
+    # A default exists for convenience in tests. Non-test code is expected to explicitly set tag.
+    tag: pd.Series = dataclasses.field(
+        default_factory=lambda: _EMPTY_TAG_SERIES.reset_index(CommonFields.LOCATION_ID, drop=True)
+    )
 
-    # TODO(tom): find a way to expose all tags in this class or maybe add `attributes` as tags with
-    # a real date.
+    @property
+    def provenance(self) -> Dict[str, str]:
```

Comment:
I think eventually this could return a typed pydantic object (that shows up in the api)

Reply:
It can be whatever you want, as long as it is a read-only view of `tag` :-P